saged/keep_ratios.py

```diff
@@ -196,8 +208,15 @@ def subset_to_equal_ratio(train_data: datasets.LabeledDataset,
             supervised_model.free_memory()
 
             accuracy = sklearn.metrics.accuracy_score(predictions, true_labels)
```

Comment:
Should this be:
```suggestion
            accuracy = sklearn.metrics.accuracy_score(true_labels, predictions)
```
(I don't think it matters to the output since accuracy should be symmetric, but probably good to be consistent with the other metrics)